packages/fsengagement/src/EngagementComp.tsx

```diff
@@ -455,11 +456,13 @@ export default function(
       return this.renderBlock(item);
     }
     renderHeaderName = () => {
-      const name = this.props.headerName || '';
+      const { json, headerName } = this.props;
+      const name = headerName || '';
+      const headerTitleStyle = json && json.headerTitleStyle || {};
       const comma = name ? ', ' : '';
       return (
-        <Text style={styles.headerName}>
-          HELLO{comma}{name.toUpperCase()}
+        <Text style={[styles.headerName, headerTitleStyle]}>
+          Hello{comma}{name}
```

Comment:
While this does look good overall, this is changing the default behavior of the component. Adding `textTransform: uppercase` to styles.headerName should make it act the same.